sap-schemas/v2.0/README.md

```diff
@@ -587,7 +587,7 @@ Constraints:
 
 - Type: `Boolean`
 - Used at: [Schema Object](https://spec.openapis.org/oas/v2.0#schema-object)
-- Description: Property contains potentially personal data. Properties annotated with `x-sap-dpp-field-semantics` need not be additionally annotated with this extension.
+- Description: Property contains potentially personal data. Properties annotated with `x-sap-dpp-field-semantics` need not be additionally annotated with this extension. Personal data describes information which is related to any information relating to an identified or identifiable natural person (data subject). An identifiable person is one who can be identified, directly or indirectly, in particular by a reference to an identifier such as a name, an identification number, location data, an online identifier, or to one or more factors specific to the physical, physiological, genetic, mental, economic, cultural, or social identity of that natural person.
```

Comment:
"information related to information relating to" - shorten as proposed for the first file